packages/client/components/NewParabolTaskButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import Icon from './Icon'
+import {ZIndex} from '~/types/constEnums'
+import FloatingActionButton from './FloatingActionButton'
+
+const Button = styled(FloatingActionButton)({
+  color: '#fff',
+  padding: '10px 12px',
+  width: '150px',
+  top: '85%',
+  left: '174%',
```

Comment:
the only difference btwn this and `NewJiraIssueButton` is this `left` value and the `<StyledLabel>` text. i think making an absolute position an argument is not so helpful , but this is might be ripe for DRY in the future.